### PR TITLE
Check calendar URI length before creation

### PR DESCRIPTION
**Pull request by miaulalala**

### CODE REVIEW

File: apps/dav/lib/CalDAV/CalDavBackend.php [hunk 1]
```diff
@@ -40,6 +40,7 @@
 
 use DateTime;
 use DateTimeInterface;
+use OC\DB\Exceptions\DbalException;
```
Comment (nickvergessen) [inline apps/dav/lib/CalDAV/CalDavBackend.php:43]: not used

**Review by nickvergessen — APPROVED** [review]

**Review by ChristophWurst — APPROVED** [review]

**Review by tcitworld — APPROVED** [review]

**Pull request merged**